Files/Views/Settings.xaml.cs

```diff
@@ -69,6 +74,15 @@ private void GoBack()
             }
         }
 
+        private void CoreDispatcher_AcceleratorKeyActivated(CoreDispatcher sender, AcceleratorKeyEventArgs e)
+        {
+            if (!e.Handled && e.VirtualKey == VirtualKey.Escape)
```

Comment:
Can you change it to recognize the back key instead?

Reply:
Ok I do it tomorrow.